src/components/typography/Heading.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from "classnames";
+import { nullValuesAsUndefined } from "null-as-undefined";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/core/_typography.scss";
+
+/**
+ * The level of heading for {@link Heading}.
+ */
+export enum HeadingLevels {
+  H1 = 1,
+  H2 = 2,
+  H3 = 3,
+  H4 = 4,
+  H5 = 5,
+  H6 = 6
+}
+
+/**
+ * The proptypes for {@link Heading}.
+ */
+export interface HeadingProps {
+  id?: string | null;
+  className?: string | null;
+  level: HeadingLevels;
+  /**
+   * Style large with a light font.
+   *
+   * See `.lbh-heading-large-light` in `lbh-frontend` for the implementation
+   * details.
+   */
+  largeLight?: boolean | null;
```

Comment:
@LBHELewis Whats the purpose of this class? So we can document it better.

Reply:
@CJWaszczuk It's just a lighter-weight heading style defined in our Style Guide. I haven't actually seen it in use anywhere and the reason for its existence hasn't been documented by the designer, so I'm happy for you to omit at this stage if that is easier?

Reply:
Ok, if that's the case, we'll drop it for now. Thanks for the quick response! :)